src/utils/AuthUtils.js

```diff
@@ -25,10 +25,33 @@ class AuthUtils {
     return await firebase.auth().signInWithPopup(this.provider_);
   }
 
-  async signOut() {
+  static async signOut() {
     return await firebase.auth().signOut();
   }
 
+  static getUser() {
+    return AuthUtils.user;
+  }
+
+  static setUser(user) {
+    AuthUtils.user = user;
+  }
+
+  static async getAuthHeader() {
+    if (!AuthUtils.user) {
+      return 'Bearer ';
+    }
+
+    if (AuthUtils.idToken) {
+      return `Bearer ${AuthUtils.idToken}`;
+    }
+
+    const idToken = await AuthUtils.user.getIdToken();
+    AuthUtils.idToken = idToken;
+
+    return `Bearer ${idToken}`;
+  }
+
   getFirebaseAuth() {
```

Comment:
This file is still using the firebase identity object, could we change it in the future so that user don't need to go to the console and download the firebase identify object and then come back and change this AuthUtils.js file?